Code/Engine/RendererCore/Meshes/Implementation/MeshBufferResource.cpp

```diff
@@ -505,6 +505,8 @@ ezResult ezMeshBufferResourceDescriptor::RecomputeNormals()
 
 ezMeshBufferResource::ezMeshBufferResource()
   : ezResource(DoUpdate::OnAnyThread, 1)
+  , m_uiPrimitiveCount(0)
```

Comment:
Would be even better to initialize in the header directly

Reply:
Indeed for constants that would be nicer. If you have the time to change that, please do, though I would accept the PR this way as well.

Reply:
I wonder whether this is something that we could use Clang-tidy for? @Ingrater ?

Reply:
I've initialized class members directly in header.